src/sentry/tasks/merge.py

```diff
@@ -228,6 +229,50 @@ def rehash_group_events(group_id, transaction_id=None, **kwargs):
     delete_group.delay(group.id)
 
 
+def decrement_new_issue_count(group):
+    """
+    Updates the new issue/groups count
+
+    Should be called with the old group from merge_group.
+    That will be deleted, and so the count is decremented by 1.
+    """
+    from sentry.models import GroupRelease, Release, ReleaseProject, ReleaseProjectEnvironment
+    release_version = group.get_first_release()
+    if release_version is None:
+        return
+    try:
+        release = Release.objects.get(version=release_version)
+    except Release.DoesNotExist:
+        return
+    release.update(new_groups=release.new_groups - 1)
```

Comment:
You'll want to use an [`F` object](https://docs.djangoproject.com/en/1.8/ref/models/expressions/#f-expressions) here to avoid a race condition if another process updates this release before this query is made.